docs/_articles/en/v3.x/salesforce-graph-engine/try-it-yourself.md

```diff
@@ -3,4 +3,112 @@ title: 'Try it yourself'
+
+It may be advantageous for you to skim those files now.
+
+### Basic Run
+Let's start with a basic evaluation of all files. `cd` into `test/code-fixtures/projects/sfge-working-app`, then run the
+following command:
+```
+sfdx scanner:run:dfa --target './force-app/main/default/classes' --projectdir './force-app/main/default' --format csv
+```
+
+Take a look at the results. Note that, as explained above, each violation has a source and a sink vertex. The source vertex
+is the start of the path in question, and the sink vertex is the point where the DML operation is occurring. If there is
+insufficient CRUD/FLS validation between those two points, a violation is thrown.
+
+Let's pick a few of those violations and look a little bit closer at them.
+
+Several methods in `AuraEnabledFls.cls` threw violations.
+
+Let's start with [`flsHelperGivenIncorrectObjctType()`](https://github.com/forcedotcom/sfdx-scanner/blob/dev-3/test/code-fixtures/projects/sfge-working-app/force-app/main/default/classes/AuraEnabledFls.cls#L4).
+Note that this method has no branches. Instead, it's just a single path all the way through. Also note that we're performing
+CRUD/FLS on the wrong object type, hence the violation. The source vertex is the line where the method is declared, and
+the sink vertex is the line where the account is inserted.
+
+Next, let's look at a slightly more complicated example, [`flsInIfBranchOnly()`](https://github.com/forcedotcom/sfdx-scanner/blob/dev-3/test/code-fixtures/projects/sfge-working-app/force-app/main/default/classes/AuraEnabledFls.cls#L60).
+This method has an `if` statement, and therefore has two paths: one that goes through the `if`, and one that doesn't. Since
+CRUD/FLS only occurs in one of those paths, a violation is thrown.
+
+Finally, let's look at a method that didn't throw a violation: [`flsDoneCorrectly()`](https://github.com/forcedotcom/sfdx-scanner/blob/dev-3/test/code-fixtures/projects/sfge-working-app/force-app/main/default/classes/AuraEnabledFls.cls#L76).
+Note that all of the fields being inserted are first being checked with the `FlsHelperClass` instance.
+
+### Running against a single file
+As you fix the problems in this file, you'll probably want to run the analyzer against this file specifically rather than
+the entire codebase. You can do that by running the following command:
+```
+sfdx scanner:run:dfa --target './force-app/main/default/classes/AuraEnabledFls.cls' --projectdir './force-app/main/default' --format csv
+```
+Please note the following, as they may save you some grief in the future.
+- The `target` file is the one containing the source vertices you wish to scan.
+- The `projectdir` must still be the entire project, so that paths can be properly built.
+
+Note that the analyzer ran faster than it did last time, because it was analyzing a smaller number of paths. And note that
+the results are only those in the targeted file.
```

Comment:
Should we say that the results have `source` only in the targeted file? Our example happens to have source and sink in the same file. If the sink was in a different file, we would've still thrown the exception since the entry method was in `--target` file.